README.md

```diff
@@ -11,8 +11,8 @@ const $ = require('sanctuary-def');
 ```
 
 The next step is to define an environment. An environment is a list of
-[types](#types) such as `$.Number` and `$.String`. `$.env` is a default
-environment, which may be augmented with custom types:
+[types](#types) such as `$.Number` and `$.String`. [`$.env`](#env) is a
+default environment, which may be augmented with custom types:
```

Comment:
Good question. When I see _the default value is `42`_ I assume that if I don't provide a value, `42` will be used. In this case, omitting the environment argument is not an option. `$.env` is simply a value which is near at hand which can be provided as the environment argument. It's not really a default at all, I suppose. Perhaps we should simply write _`$.env` is one possible environment_. What do you think?